src/index.html

```diff
@@ -8,9 +8,51 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
+
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="#" 
+      method="get"
+      data-qa="big"
+      class="form_search1" 
```

Comment:
form_search1 is not the best className, how about form_search and maybe form_search--big if you had to add some specific css rules for form which contains bigger input?